### PR TITLE
fix: add share link

### PR DESCRIPTION
**Pull request by lelenei**
closes #

### CODE REVIEW

File: packages/mask/src/plugins/Gitcoin/SNSAdaptor/DonateDialog.tsx [hunk 1]
```diff
@@ -131,7 +133,7 @@ export function DonateDialog(props: DonateDialogProps) {
                           isTwitter(activatedSocialNetworkUI) || isFacebook(activatedSocialNetworkUI)
                               ? `Follow @${
                                     isTwitter(activatedSocialNetworkUI) ? t('twitter_account') : t('facebook_account')
-                                } (mask.io) to donate Gitcoin grants.`
+                                } (mask.io) to donate Gitcoin grants.${postLink}`
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Gitcoin/SNSAdaptor/DonateDialog.tsx:136]:
```ts
{
    '#mask_io',
    postLink,
}
```

File: packages/mask/src/plugins/Gitcoin/SNSAdaptor/DonateDialog.tsx [hunk 2]
```diff
@@ -134,7 +136,9 @@ export function DonateDialog(props: DonateDialogProps) {
                                 } (mask.io) to donate Gitcoin grants.`
                               : ''
                       }`,
+
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Gitcoin/SNSAdaptor/DonateDialog.tsx:139]:
```suggestion
```

**Comment by guanbinrui** [comment]
@DimensionDev/qa

**Comment by wenluomask** [comment]
Please merge develop into your branch, then I can test in a cheaper way.

**Review by guanbinrui — APPROVED** [review]

**Comment by wenluomask** [comment]
I don't find any way to share right now. ><

**Pull request merged**